build.sbt

```
@@ -14,6 +14,16 @@ val commonSettings: Seq[Setting[_]] = Seq(
   organization := "org.scala-js",
   scalacOptions ++= Seq("-deprecation", "-feature", "-Xfatal-warnings"),
 
+  scalacOptions ++= {
+    if (isSnapshot.value)
+      Seq.empty
+    else {
```

In our coding style, if one of the two branches of the if needs braces, then the other one must have braces as well:
```suggestion
    if (isSnapshot.value) {
      Seq.empty
    } else {
```